src/main/resources/templates/db_scripts/schema.json

```diff
@@ -178,6 +178,10 @@
           "tOps" : "ADD"
         }
       ]
+    },
+    {
+      "tableName" : "patron_block_conditions",
+      "withMetadata" : true
```

Comment:
Can you add "fromModuleVersion" so that this table gets created on upgrades?

Reply:
added